### PR TITLE
update README

### PR DESCRIPTION
**Pull request by unseenmagik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,28 +13,51 @@ Simple app displaying a leaderboard of Pokémon Go trainers.
...
 npm ci
 cp config/env/config.example config/env/production.json
 ```
 
+### Configuration:
 Then edit the file `config/env/production.json` to put your configuration.
 
-## Running the application
+### Discord Auth
+* Create a [discord bot](https://discord.com/developers) and fill in the config options below. All variable require a value.
```
Comment (na-ji) [inline README.md:32]: > All variable require a value
I've now fixed an issue with that. Now, only the `clientId` and the `clientSecret` are required. The `botToken` is required if the `guildId` or the `roleId` are filled. If not used, the `guildId`/`roleId` should be empty arrays.

Comment (unseenmagik) [reply README.md]: I have not updated this in the readme

Comment (lenisko) [reply README.md]: guildId should be available with oauth alone, but those are small details

Comment (na-ji) [reply README.md]: Here the guildId is a filter. It's to be sure that the person login in through oauth2 is a member of your server.

**Review by na-ji — APPROVED** [review]

**Pull request merged**